zerver/tests/test_markdown.py

```diff
@@ -1813,6 +1814,20 @@ def test_default_code_block_language(self) -> None:
         with_language, without_language = re.findall(r"<pre>(.*?)$", rendered, re.MULTILINE)
         self.assertFalse(with_language == without_language)
 
+    def test_mention_topic_wildcard(self) -> None:
+        user_profile = self.example_user("othello")
+        msg = Message(sender=user_profile, sending_client=get_client("test"))
+
+        for topic_wildcard in topic_wildcards:
+            content = f"@**{topic_wildcard}** test"
+            rendering_result = render_markdown(msg, content)
+            self.assertEqual(
+                rendering_result.rendered_content,
+                f'<p><span class="user-mention" data-user-id="*">@{topic_wildcard}</span> test</p>',
```

Comment:
Hmm, can you start an "api design" conversation about this `data-user-id` value? I think it's possible we want a special value of "topic" or some other indication that is the a topic mention; I guess simplest would be to keep the `data-user-id` as `*` but also have a `topic-mention` class on the `user-mention` element. As it is, the only way for clients to determine whether a given mention inside a message was a topic mention is by parsing the text inside, and we definitely don't want to use that mechanism, since it's not at all robust for future changes to what syntax we use.